scripts/diff-types.js

```diff
@@ -0,0 +1,72 @@
+// module.exports = async ({
+//   core
+// }) => {
+
+const fs = require('fs');
+const yaml = require('yaml');
+
+let tutoneConfig = null
+let schema = null
+
+try {
+  const tutoneConfigFile = fs.readFileSync('.tutone.yml', 'utf8')
+  tutoneConfig = yaml.parse(tutoneConfigFile)
+
+  const schemaFileLatest = fs.readFileSync('schema.json', 'utf8');
+  schema = JSON.parse(schemaFileLatest);
+} catch(err) {
+  console.error(err);
+}
+
+const package = tutoneConfig.packages.find(x => x.name === 'entities');
+// const packageTypesByName = package.types.filter(t => !t.skip_type_create && t.name.includes("Entity")).map(t => t.name);
+const schemaTypes = schema.types.filter(t => t.name.endsWith("Entity") || t.name.endsWith("EntityOutline")).map(t => t.name)
+
+// const typesDiff = schemaTypes.filter(x => {
+//   console.log("schema type:", x);
+//   console.log("pkg type:   ", packageTypesByName.includes(x));
+
+//   return !packageTypesByName.includes(x)
+// });
+
+// console.log('packageTypes:', JSON.stringify(packageTypesByName, null, 2));
+// console.log('schemaTypes: ', JSON.stringify(schemaTypes, null, 2));
+// console.log('typesDiff:   ', typesDiff);
+// console.log('typesDiff:   ', typesDiff.length);
+
+
+const packageTypes = getPackageTypesFromTypesFile('/Users/sblue/dev/newrelic-client-go/pkg/entities/types.go');
+const packageTypesLatest = getPackageTypesFromTypesFile('/Users/sblue/Desktop/2024.06.04-entities-package-types-latest.json');
+
```

Comment:
These need to be updated to point to the correct path.

Reply:
Oh right. We actually won't need this `diff-types.js` file. That was a script most to just gather information about what the differences were and why. I'll remove the file. Thanks for the reminder.